content/en/serverless/step_functions/enhanced-metrics.md

```diff
@@ -3,7 +3,7 @@ title: Enhanced metrics for AWS Step Functions
 kind: documentation
 ---
 
-Datadog generates enhanced metrics for AWS Step Functions, similarly to [enhanced metrics for AWS Lambda][1]. Enhanced Step Functions metrics are distinguished by being in the `aws.states.enhanced.*` namespace.
+In addition to [ingesting integration metrics from AWS][3], Datadog generates enhanced metrics for AWS Step Functions, similar to [enhanced metrics for AWS Lambda][1]. Enhanced Step Functions metrics are distinguished by being in the `aws.states.enhanced.*` namespace. To add enhanced metrics, follow the [AWS Step Function monitoring installation instructions][3] and ensure that `DD_ENHANCED_METRICS_ENABLED` is set to `true`. 
```

Comment:
It should be "DD_ENHANCED_METRICS" instead of "DD_ENHANCED_METRICS_ENABLED". It is consistent with Lambda. The term "DD_ENHANCED_METRICS_ENABLED" is only used in source codes for easier understanding of codes

Reply:
fixed